scripts/changelog.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+#
+# Copyright 2017 The rkt Authors
+# Copyright 2020 Kinvolk
+#
+# Create a changelog.
+#
+# The env variable RANGE specifies the range of commits to be searched for the changelog.
+# If unset the latest tag until origin/master will be set.
+#
+# The env variable GITHUB_TOKEN can specify a GitHub personal access token.
+# Otherwise one could run into GitHub rate limits. Go to
+# https://github.com/settings/tokens to generate a token.
+
+set -e
+
+jq --version >/dev/null 2>&1 || {
+    echo "could not find jq (JSON command line processor), is it installed?"
+    exit 255
+}
+
+if [ -z "${RANGE}" ]; then
+    LATEST_TAG=$(git describe --abbrev=0)
```

Comment:
I added tag v0.0.0 on some previous commit for testing.

To get this line to work, git asked me to add `--tags`:
```sh
$ scripts/changelog.sh 
fatal: No annotated tags can describe 'b6fd0329486950accb7f42ad7e80824623d6635c'.
However, there were unannotated tags: try --tags.
```

Maybe we should compare last tag with HEAD? Or maybe we should create changelog after creating release tag?

Reply:
The problem is that the `v0.0.0` tag you've created is not annotated (and they should be if you follow the release process). I can add `--tags` even though we shouldn't have non-annotated tags, it doesn't hurt.

Reply:
Ah, good to know. Thanks.